src/store/api.js

```diff
@@ -101,8 +101,28 @@ export async function getPaginatedEvents({
   return { items: drops, total };
 }
 
-export async function getEvent(id) {
-  return await fetchPOAPApi(`/events/id/${id}`);
+export async function getDrop(id) {
```

Comment:
does it make sense to specify what is that getDrop returns?

Reply:
This is JavaScript, not TypeScript. Or you mean adding jsdoc?

Reply:
Oh my bad. Then its ok